test/integration/globals/error.js

```diff
@@ -18,6 +18,40 @@ context('Global Error Page', function() {
       .should('not.exist');
   });
 
+  // NOTE: 'legacy' means the workspace name isn't included in the URL
+  specify('404 not found - legacy routes', function() {
+    cy
+      .visit('/404', { isRoot: true });
```

Comment:
really I think this and the above test should be combined and it should by something like `visit('/route-does-not-exist')` instead of `/404`.

Reply:
I attempted that, but couldn't get coverage for these lines: https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/apps/globals/error_app.js#L50-L60.

That code wouldn't run if the workspace name is included in the url on the initial load (i.e. `/one/worklist/owned-by` vs. `/worklist/owned-by`).

But maybe I missed something, so I'll look at these tests again.

Reply:
oh right.. yeah so it would be need to be `visit` with `{ isRoot: true }`.. are we testing legacy routes somewhere else and this is testing for a non-legacy route (the else)?  regardless we should not use `/404` for this as it's confusing.  We're answering "can I go to the route or is it a 404?"  So "can I go to 404 or is it a 404?" is 😬

Reply:
I just updated those 404 tests to load an initial route of `/route-does-not-exist` instead of `/404`.

My understanding is the `'404 not found'` test in this file checks the non-legacy route. And the `'404 not found - legacy routes'` test I created in this PR checks the legacy route, where `{isRoot: true}` is added.